diff --git a/compiling/makefile b/compiling/makefile
index 8cfed1b..8007155 100644
--- a/compiling/makefile
+++ b/compiling/makefile
@@ -13,8 +13,8 @@
 #================================================================================#
 
    # 32-bit or 64-bit?
-BITS = 32
-#BITS = 64
+#BITS = 32
+BITS = 64
 
 
    # Location of source files for TurbSim and its dependencies.
@@ -42,7 +42,8 @@ endif
 FC      = gfortran
 
 FFLAGS  = -O2 -m$(BITS) -fbacktrace -ffree-line-length-none -x f95-cpp-input
-LDFLAGS = -O2 -m$(BITS) -fbacktrace -Wl,--large-address-aware
+# LDFLAGS = -O2 -m$(BITS) -fbacktrace -Wl,--large-address-aware
+LDFLAGS = -O2 -m$(BITS) -fbacktrace
 
 #FFLAGS  = -O0 -m$(BITS) -fbacktrace -ffree-line-length-none -x f95-cpp-input -g -pg
 #LDFLAGS = -O0 -m$(BITS) -fbacktrace -Wl,--stack=999999999,--large-address-aware -g -pg
